scripts/benchmarking/README.md

```diff
@@ -52,18 +52,16 @@
 
 ### Index
 
-- <span><b>chairman.sh</b></span> <br/>  Run the Chairman consensus checker against a cluster started
-                     by `shelley-testnet*.sh` family of scripts
```

Comment:
What's the reason for deleting this line?

Reply:
The chairman script being referred to was very outdated. I want to set a clear boundary between what scripts the benchmarking team use (and therefore should maintain) and what the node team use.